{"name": "Неуспешный логин с неверными данными", "status": "passed", "start": 1733245287873, "stop": 1733245288177, "uuid": "221814fa-04ba-4d75-b1cc-c18f066218ba", "historyId": "875475173a8d8fcaadc0586d31e090c2", "testCaseId": "875475173a8d8fcaadc0586d31e090c2", "fullName": "tests.test_login.TestLogin#test_login_failure", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "96905-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}